templates/rhpam73-authoring-ha.yaml

```diff
@@ -318,17 +318,17 @@ parameters:
   required: true
 ## OpenShift Enhancement BEGIN
 - displayName: Prefer KIE Server OpenShift Service [Tech Preview]
-  description: Enables connection to KIE Server via OpenShift internal Service endpoint (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property)
+  description: If OpenShift Integration of the Business Central is turned on, set this parameter to true to enables connection to KIE Server via an OpenShift internal Service endpoint. Otherwise the Business Central uses a Route to connect to KIE Server. (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property)
```

Comment:
**Original text:**
If OpenShift Integration of the Business Central is turned on, set this parameter to true to enables connection to KIE Server via an OpenShift internal Service endpoint. 

**Suggested changes:**
If OpenShift **i**ntegration of **Business Central** is turned on, **by setting this parameter to true enables** connection to KIE Server via an OpenShift internal Service endpoint.